src/main/java/frc/robot/RobotContainer.java

```diff
@@ -110,36 +112,71 @@ public RobotContainer(RobotBase robot) {
    */
   
   private void configureBindings() {
-      Joystick leftJoystick = new Joystick(0);
-      Joystick rightJoystick = new Joystick(1);
-      Joystick altJoystick = new Joystick(2);
-
-
-      swerveDrive.setDefaultCommand(new RunCommand(() -> {
-        if (robot.isTeleopEnabled()){
-          //drive
-          swerveDrive.drive(
-            limitX.calculate(applyDeadband(-leftJoystick.getX(), DrivetrainConstants.DRIFT_DEADBAND))*DriverConstants.speedMultiplier,
-            limitY.calculate(applyDeadband(-leftJoystick.getY(), DrivetrainConstants.DRIFT_DEADBAND))*DriverConstants.speedMultiplier,
-            applyDeadband(-rightJoystick.getX(), DrivetrainConstants.ROTATION_DEADBAND)*DriverConstants.angleMultiplier);
-        }
-        else 
-        {
-          swerveDrive.drive(0,0,0);
-        }
-        
-      }, swerveDrive));
+    Joystick leftJoystick = new Joystick(0);
+    Joystick rightJoystick = new Joystick(1);
+    Joystick altJoystick = new Joystick(2);
+
+    frontLeftWheel.getEncoder().setInverted(true);
+    frontRightWheel.getEncoder().setInverted(true);
+    backLeftWheel.getEncoder().setInverted(true);
+    backRightWheel.getEncoder().setInverted(true);
+
+    backRightWheel.getEncoder().setZeroOffset(DrivetrainConstants.BACK_RIGHT_ENCODER_OFFSET);
+    backLeftWheel.getEncoder().setZeroOffset(DrivetrainConstants.BACK_LEFT_ENCODER_OFFSET);
+    frontRightWheel.getEncoder().setZeroOffset(DrivetrainConstants.FRONT_RIGHT_ENCODER_OFFSET);
+    frontLeftWheel.getEncoder().setZeroOffset(DrivetrainConstants.FRONT_LEFT_ENCODER_OFFSET);
+
+    frontLeftWheel.getEncoder().setInverted(true);
+    frontRightWheel.getEncoder().setInverted(true);
+    backLeftWheel.getEncoder().setInverted(true);
+    backRightWheel.getEncoder().setInverted(true);
+
+    coastDrive();
+
+    swerveDrive.setDefaultCommand(new RunCommand(() -> {
+      if (robot.isTeleopEnabled()){
+        //drive
+        swerveDrive.drive(
+          limitY.calculate(applyDeadband(-leftJoystick.getY(), DrivetrainConstants.DRIFT_DEADBAND))*DriverConstants.speedMultiplier,
+          limitX.calculate(applyDeadband(-leftJoystick.getX(), DrivetrainConstants.DRIFT_DEADBAND))*DriverConstants.speedMultiplier,
+          applyDeadband(-rightJoystick.getX(), DrivetrainConstants.ROTATION_DEADBAND)*DriverConstants.angleMultiplier);
+          // 0,0,0);
```

Comment:
Clean up code

Reply:
Ok, it's kind of convenient just having it there but if it's bad form I can take it out

Reply:
Not a huge deal, just looks kinda messy. I would probably take it out to commit, then put it back in my local code, but it's up to you.